qa/L0_batcher/batcher_test.py

```diff
@@ -38,6 +38,10 @@
 
 import tritonclient.grpc as grpcclient
 
+# By default, find tritonserver on "localhost", but can be overridden
+# with TRITONSERVER_IPADDR envvar
+_tritonserver_ipaddr = os.environ.get('TRITONSERVER_IPADDR', 'localhost')
```

Comment:
We would need to make this change for the gitlab scripts that are used for windows testing as well. For example L0_onnx_execution_provider and L0_openvino_models?

Reply:
The gitlab MR should have all the necessary changes.